### DISCUSSION TITLE
Are preview APIs published?

### DISCUSSION THREAD

**Discussion by TheMoti** [ceph/go-ceph#820] [General]
I wanted to use an API which is still in preview mode, by importing the lib and was wondering if it exists in pre-built go packages to use.

**Comment by phlogistonjohn** [comment]
You can use preview APIs by using the `ceph_preview` build tag.  But, as the name implies, the tags apply when you are building from source. I don't know what "pre-built go packages" you are referring to.

Here' a link about build tags in general: https://www.programming-books.io/essential/go/conditional-compilation-with-build-tags-d1980344374d45c082c914c2aafa50cf

If you have any additional questions about build tags or how we use them, just ask!
